src/main/java/com/example/springboot/service/CsvParserService.java

```diff
@@ -0,0 +1,8 @@
+package com.example.springboot.service;
+
+import com.example.springboot.dto.ReviewListDto;
+import java.util.List;
+
+public interface CsvParserService {
+    ReviewListDto parseCsvRecords(List<String> records);
```

Comment:
```suggestion
    ReviewDto parseLine(String line);
```
I think this will be more reusable
we will be able to call this method for each line read from the file

Reply:
Done